tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h

```diff
@@ -40,21 +40,21 @@ using dnnl::memory;
 using dnnl::prop_kind;
 using dnnl::stream;
 
-using EltwiseFwdPd = dnnl::eltwise_forward::primitive_desc;
+using EltwiseFwdActivationPd = dnnl::eltwise_forward::primitive_desc;
 
 namespace tensorflow {
 
 template <typename T>
-class MklEltwiseFwdParams {
+class MklEltwiseFwdActivationParams {
```

Comment:
It's not just the class name. The implementation of this class is very similar to `MklEltWiseFwdParams` in `mkl_relu_op.cc`. I'm okay with renaming as a quick fix for TF 2.10 cherry-pick. But long-term I think we should just consolidate them to one class.

Please add this comment in the code (please wrap the line if it's longer than 80 characters):
```
// TODO(tf-onednn): Consolidate this class with `MklEltWiseFwdParams` in `mkl_relu_op.cc`.
```

Thank you very much!

Reply:
Thanks @penpornk and @sachinmuradi. Added comment as requested.